demo/Alsa cpp/main.cpp

```diff
@@ -0,0 +1,216 @@
+/*
+ * 
+ * Project: Porcupine Alsa
+ * Created Date: Tuesday, September 18th 2018, 7:41:55 pm
+ * Author: Greg
+ * -----
+ * Last Modified: Tue Sep 18 2018
+ * Modified By: Greg
+ * -----
+ * Copyright (c) 2018 Greg
+ * 
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to
+ * deal in the Software without restriction, including without limitation the
+ * rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+ * sell copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ * 
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN
+ * AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+ * WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ * 
+ */
+
+
+#include <iostream>
+#include <alsa/asoundlib.h>
+#include "../../include/pv_porcupine.h"
```

Comment:
A cleaner way could be to add a -I in compile line with the path to include directory. But this would work as well.